<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnProcessBool">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bool Jaccard</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtWord">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProcessDistance">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal Jacc.</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblResults">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>441</width>
     <height>711</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="lblAlert">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Resultado: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
